src/thing_shots.c

```diff
@@ -661,16 +661,15 @@ long shot_hit_object_at(struct Thing *shotng, struct Thing *target, struct Coord
     HitPoints damage = 0;
     if (shotng->shot.damage)
     {
-        if (object_can_be_damaged(target)) // do not damage objects that cannot be destroyed
```

Comment:
was undoing your object_can_be_damaged change intentional?

Reply:
Undid that removal, so it should be good now